src/types.rs

```diff
@@ -122,6 +122,22 @@ pub struct HostStats {
     pub timestamp: Option<i64>,
 }
 
+// Type for the list/available endpoint
+#[derive(Serialize, Deserialize, Clone)]
+#[serde(crate = "rocket::serde")]
+#[serde(rename_all = "camelCase")]
+pub struct HostLatest {
+    #[serde(rename = "_id")]
+    pub _id: String,
+    pub holo_network: Option<String>,
+    pub channel: Option<String>,
+    pub holoport_model: Option<String>,
+    pub ssh_status: Option<bool>,
+    pub zt_ip: Option<String>,
+    pub wan_ip: Option<String>,
+    pub timestamp: Option<i64>,
+}
+
```

Comment:
Can you explain to me what is the reason to use '_id' instead of `holoportId` in the return type? It seems a little bit weird to have field named `_id`, because `_` usually denotes local/internal field.

Reply:
When you do a mongodb pipeline aggregation, you choose a field to be the new `_id` of the aggregation. In our case, we choose to aggregate all records where the `holoportId` is the same, and the result of the aggregation is automatically an aggregation where the _id is the `holoportId`.

So if we want to rename `_id`, then I think we need to do some additional data transformation which, while not hard, seems unnecessary to me.

Reply:
@peeech 

Here's what the aggregation pipeline looks like, if you look at the bottom row you'll see that the _ids are the holoportIds

![image](https://user-images.githubusercontent.com/16625137/157278714-182f4dbb-0eb6-42ca-bad4-f896ca376930.png)